clonevirtualenv.py

```diff
@@ -291,8 +291,8 @@ def main():
     except ValueError:
         print("virtualenv-clone {}".format(__version__))
         parser.error("not enough arguments given.")
-    old_dir = os.path.normpath(os.path.abspath(old_dir))
-    new_dir = os.path.normpath(os.path.abspath(new_dir))
+    old_dir = os.path.realpath(os.path.normpath(os.path.abspath(old_dir)))
+    new_dir = os.path.realpath(os.path.normpath(os.path.abspath(new_dir)))
```

Comment:
it seems that `os.path.realpath` is all that is actually needed as it expands to the full, canonical path and the usage of both `normpath` and `abspath` can be removed.

```
In [10]: x
Out[10]: 'foodir2'

In [11]: os.path.islink(x)
Out[11]: True

In [12]: os.path.realpath(x)
Out[12]: '/Users/xxxxxx/Workspace/projects/xxxxxxxxxxx/foodir'
```

so this could just be:

```
old_dir = os.path.realpath(old_dir)
new_dir = os.path.realpath(new_dir)
```

do you agree?

Reply:
@benrudolph can you update this to remove the unnecessary path code here?

Reply:
Oops, I was looking at just the first commit, and you changed this in a second one.  Merged, thanks @benrudolph!